docs/docs/genai/tracing/quickstart/index.mdx

```diff
@@ -0,0 +1,137 @@
+
+## Dependency
+
+To connect your GenAI application to the MLflow server, you will need to install the MLflow client SDK.
+
+<TabsWrapper>
+<Tabs>
+<TabItem value="python" label="Python(OpenAI)" default>
+```bash
+pip install --upgrade mlflow openai>=1.0.0
+```
+</TabItem>
+
+<TabItem value="typescript" label="Typescript(OpenAI)" default>
+```bash
+npm install mlflow-openai
+```
+</TabItem>
+</Tabs>
+</TabsWrapper>
+
+:::info
+
+While this guide features an example using the OpenAI SDK, the same steps apply to other LLM providers, including Anthropic, Google, Bedrock, and many others.
+
+For a comprehensive list of LLM providers supported by MLflow, see the <ins>[LLM Integrations Overview](/genai/tracing/integrations)</ins>.
+
+:::
+
+## Start Tracing
+
+Once your experiment is created, you're ready to connect to the MLflow server and begin sending traces from your GenAI application.
+
+<TabsWrapper>
+<Tabs>
+<TabItem value="python" label="Python(OpenAI)" default>
+
+```python
+import mlflow
+from openai import OpenAI
+
+# Specify the tracking URI for the MLflow server.
+mlflow.set_tracking_uri("http://localhost:5000")
+
+# Specify the experiment you just created for your GenAI application.
+mlflow.set_experiment("My Application")
+
+# Enable automatic tracing for all OpenAI API calls.
+mlflow.openai.autolog()
+
+client = OpenAI()
+# The trace of the following is sent to the MLflow server.
+client.chat.completions.create(
+    model="o4-mini",
+    messages=[
+        {"role": "system", "content": "You are a helpful weather assistant."},
+        {"role": "user", "content": "What's the weather like in Seattle?"},
+    ],
+)
+```
+
+</TabItem>
+
+<TabItem value="typescript" label="Typescript(OpenAI)">
+
+```typescript
+import { init } from "mlflow-tracing";
+import { tracedOpenAI } from "mlflow-openai";
+import { OpenAI } from "openai";
+
+init({
+    trackingUri: "http://localhost:5000",
+    // NOTE: specifying experiment name is not yet supported in Typescript SDK.
+    // You can get the experiment id from the URL when viewing experiment details on the MLflow UI.
+    experimentId: "<experiment-id>",
+});
+
+// Wrap the OpenAI client with the tracedOpenAI function to enable automatic tracing.
+const client = tracedOpenAI(new OpenAI());
+
+// The trace of the following is sent to the MLflow server.
+client.chat.completions.create({
+    model: "o4-mini",
+    messages: [
+        {"role": "system", "content": "You are a helpful weather assistant."},
+        {"role": "user", "content": "What's the weather like in Seattle?"},
+    ],
+})
+```
+
+</TabItem>
+</Tabs>
+</TabsWrapper>
+
+## View Your Traces on the MLflow UI
+
+After running the code above, go to the MLflow UI and select the "My Application" experiment, and then select the "Traces" tab. It should show the newly created trace.
+
+<ImageBox src="/images/llms/tracing/quickstart/single-openai-trace-list.png" alt="Single Trace" />
```

Comment:
For this image, could we include the whole UI like
<img width="1650" height="420" alt="image" src="https://github.com/user-attachments/assets/4048841a-fd5e-4dbb-8f0e-58cc7cefa392" />
This also gives user a first impression about what other features are available on UI